src/index.ts

```diff
@@ -445,14 +445,15 @@ function readConfig (compilerOptions: any, project: string | boolean | undefined
   delete result.config.compilerOptions.outFile
   delete result.config.compilerOptions.declarationDir
 
-  const basePath = result.path ? dirname(result.path) : cwd
+  const normalizedConfigPath = result.path && normalizeSlashes(result.path)
```

Comment:
I realized we need to handle when no `tsconfig.json` is found. Can't do replace on undefined. So fixed that to.